product_docs/docs/pgd/5/reference/functions.mdx

```diff
@@ -1141,6 +1141,14 @@ bdr.alter_commit_scope(
     rule TEXT)
 ```
 
+#### Note
```

Comment:
Don't we need this as a note?

Reply:
No, it's a note for users not a specific admonition. It will become a deprecation note at some point.